features/security/src/main/feature/feature.xml

```diff
@@ -624,4 +624,13 @@
              description="Verifies that the installation is located in a secure directory.">
         <bundle>mvn:ddf.platform.security/secure-boot/${project.version}</bundle>
     </feature>
+
+    <feature name="tink-test-dependencies" version="${project.version}"
```

Comment:
❓ It seems odd to have a test dependencies feature here. Do we do this anywhere else?

Reply:
We only need this feature for component/integration tests that need to load the tink dependencies. Let me 👀 if there is a better place to put this real quick.

Reply:
Decided to throw the feature in `<ddf_source>\features\test-utilities\src\main\feature\feature.xml` instead.